src/plugins/orangehrmPimPlugin/Api/EmployeeContactDetailsAPI.php

```diff
@@ -301,6 +304,10 @@ public function saveContactDetails(): Employee
             self::PARAMETER_OTHER_EMAIL
         );
 
+        if (!empty($workEmail) && !empty($otherEmail) && $workEmail === $otherEmail) {
+            throw $this->getBadRequestException('Work Email and Other Email Cannot Be The Same');
+        }
```

Comment:
in 4.x when trying to add two new emails that are the same `Already exists` message is shown. I feel the Bad Request approach is better since the message makes more sense. Is that okay or should I go with 4.x behaviour?

![image](https://user-images.githubusercontent.com/97493184/163972552-7c6ea25d-53b1-4108-9fa2-2669d4a9c7a6.png)

Reply:
you are validate  from BE right

Reply:
Can write a rule for the front end as well to show some custom message. But BE is needed because contact details can be added via API also